tests/test_bio_encoding.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+from flair.datasets import ColumnDataset
+import json
+from cassis import *
+from flair.models import SequenceTagger
+from flair.data import Token,Sentence
+import unittest
+
+SENTENCE_TYPE = "de.tudarmstadt.ukp.dkpro.core.api.segmentation.type.Sentence"
+TOKEN_TYPE = "de.tudarmstadt.ukp.dkpro.core.api.segmentation.type.Token"
+NER_TYPE = "de.tudarmstadt.ukp.dkpro.core.api.ner.type.NamedEntity"
+
+
+def bio_encoding(cas, sentence, tagset, t):
```

Comment:
Why is this code here? It should not be in the test folder...

Reply:
When I import the bio_encoding function from app_flair, it also needs to load the models and get the model parameters from the config file of gunicorn. But if we only run the test case, it will not load the config file of gunicorn.

Reply:
This should be prevented by the `if __name__ == "__main__":`. The else [here](https://github.com/inception-project/external-recommender-flair/blob/7093b35f04bd86bbe928451fb96cf94b8faa9684/app_flair.py#L464) looks strange, is it causing that?

Reply:
Yes. The "else" part is used when we start the application with gunicorn and load the parameters from the config file.

Reply:
Can you check whether using an `elif` with the solution in [here](https://stackoverflow.com/questions/22192519/detect-if-flask-is-being-run-via-gunicorn/25877937) works?

Reply:
Thanks. I will check it.